Std/Data/BitVec/Lemmas.lean

```diff
@@ -122,11 +122,21 @@ private theorem lt_two_pow_of_le {x m n : Nat} (lt : x < 2 ^ m) (le : m ≤ n) :
 
 /-! ### cast -/
 
-@[simp] theorem toNat_cast (e : m = n) (x : BitVec m) : (cast e x).toNat = x.toNat := rfl
+section
+variable (h : w = v) (x : BitVec w)
 
-@[simp] theorem getLsb_cast : (cast h v).getLsb i = v.getLsb i := by
-  cases h
-  simp
+@[simp] theorem toNat_cast : (cast h x).toNat = x.toNat := rfl
+@[simp] theorem toFin_cast : (cast h x).toFin = x.toFin.cast (by rw [h]) := rfl
+
+@[simp] theorem getLsb_cast : (cast h x).getLsb i = x.getLsb i := by
+  subst h ; simp
```

Comment:
```suggestion
  subst h; simp
```
I think usually no space before `;`, but probably the style guide doesn't care. One day no one will care, because a code formatter will run automatically.

Reply:
Are you sure? Other lemmas in this same file have it written with the space, which is what made me include it, too. For example:
```lean
@[simp] theorem ofNat_sub_ofNat {n} (x y : Nat) : x#n - y#n = .ofNat n (x + (2^n - y % 2^n)) := by
  apply eq_of_toNat_eq ; simp [BitVec.ofNat]

@[simp] protected theorem sub_zero (x : BitVec n) : x - (0#n) = x := by apply eq_of_toNat_eq ; simp
```

Should I change those as well, for the sake of consistency?